modbus_tk/modbus.py

```diff
@@ -177,6 +182,28 @@ def execute(
                 # slave + func + bytcodeLen + bytecode x 2 + crc1 + crc2
                 expected_length = 2 * quantity_of_x + 5
 
+        elif function_code == defines.READ_FILE_RECORD:
+            is_read_function = True 
+            if (isinstance(number_file, tuple)
```

Comment:
New line after opening parenthesis. first condition should be at the same level than others.

Reply:
Accepted, resolved in the next commits